.github/workflows/phylogenetic-ncbi.yaml

```
@@ -30,34 +34,51 @@ on:
           If set, builds will be deployed to s3://nextstrain-staging/avian-flu_trials_<trial_name>_*
         required: false
         type: string
+      config-yaml:
+        description: |
+          Currently we have H5N1/D1.1 and H5N1/cattle-outbreak (B3.13) builds available.
+          The choices here correspond to the YAML of the same name in config/
+        required: true
+        type: choice
+        options:
+          - h5n1-cattle-outbreak
+          - h5n1-d1.1
 
 jobs:
   summary:
     runs-on: ubuntu-latest
     steps:
       - name: summary_step
         run: |
-          URL_A="avian-flu/h5n1-cattle-outbreak/genome"
-          URL_B="avian-flu/h5n1-cattle-outbreak/ha"
+          if [[ "$CONFIG_NAME" = "h5n1-d1.1" ]]; then
+            URL_A="avian-flu/h5n1-d1.1/genome"
+          else
+            URL_A="avian-flu/h5n1-cattle-outbreak/genome"
+            URL_B="avian-flu/h5n1-cattle-outbreak/ha"
+          fi;
           if [[ "$TRIAL_NAME" ]]; then
               echo "### Trial Build URLs" >> $GITHUB_STEP_SUMMARY
               URL_BASE="https://nextstrain.org/staging/avian-flu/trials"
               TRIAL_NAME_URL=$( echo "$TRIAL_NAME" | sed "s|_|/|g" )
               echo "" >> $GITHUB_STEP_SUMMARY
               echo "  * ${URL_BASE}/${TRIAL_NAME_URL}/${URL_A}" >> $GITHUB_STEP_SUMMARY
-              echo "  * ${URL_BASE}/${TRIAL_NAME_URL}/${URL_B}" >> $GITHUB_STEP_SUMMARY
-              echo "  * etc" >> $GITHUB_STEP_SUMMARY
+              if [[ -v ${URL_B} ]]; then
+                echo "  * etc" >> $GITHUB_STEP_SUMMARY
+                echo "  * ${URL_BASE}/${TRIAL_NAME_URL}/${URL_B}" >> $GITHUB_STEP_SUMMARY
+              fi;
```

This conditional isn't working as intended - see the [cattle-outbreak trial build](https://github.com/nextstrain/avian-flu/actions/runs/13506833540) which (incorrectly) doesn't include these 2 lines. This will also be the case for lines 74-77 below.

---

(The order of the two `echo`s is also reversed 🤦 )

---

I believe with `-v` you have to check the variable name directly (remove `${}`):

```suggestion
              if [[ -v URL_B ]]; then
                echo "  * etc" >> $GITHUB_STEP_SUMMARY
                echo "  * ${URL_BASE}/${TRIAL_NAME_URL}/${URL_B}" >> $GITHUB_STEP_SUMMARY
              fi;
```

---

I was going to complain about bash but this actually makes perfect sense. [Fixed up](https://github.com/nextstrain/avian-flu/compare/5843d0cab6fa752e066a02bc996c717fab315b98..58d2c4bc047f11655616ae7286c8f6d78757b5ce)